scripts/linters/python_linter.py

```diff
@@ -544,6 +508,48 @@ def _lint_py_files_for_python3_compatibility(self):
             'Python linting for Python 3 compatibility finished.')
         return summary_messages
 
+    def _check_import_order(self):
```

Comment:
Why is this moved below?

Reply:
This is a third_party linter, before it was under _CustomPythonLinter_ class, so I moved it under _PythonThirdPartyLinter_ class.

Reply:
Okay makes sense.